server.js

```diff
@@ -1,27 +1,94 @@
-import cors from "cors";
-import express from "express";
-import mongoose from "mongoose";
+import cors from "cors"
+import express from "express"
+import expressListEndpoints from "express-list-endpoints"
+import dotenv from "dotenv"
+import mongoose from "mongoose"
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl);
-mongoose.Promise = Promise;
+dotenv.config()
+
+const mongoUrl =
+  process.env.MONGO_URL || "mongodb://localhost/project-happy-thoughts-api"
+mongoose.connect(mongoUrl)
+mongoose.Promise = Promise
+
+const Thought = mongoose.model("Thought", {
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+  },
+  hearts: {
+    type: Number,
+    default: 0,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now,
+  },
+})
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+const port = process.env.PORT || 8080
+const app = express()
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+app.use(cors())
+app.use(express.json())
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+  const endpoints = expressListEndpoints(app)
+  res.json(endpoints)
+})
+
+// Endpoint only get 20 thoughts sorted
+app.get("/thoughts", async (req, res) => {
+  const thoughts = await Thought.find()
+    .sort({ createdAt: "desc" })
+    .limit(20)
+    .exec()
+  res.json(thoughts)
```

Comment:
Would be nice with some error handling here as well as a status code. Consider a try/catch block